hooks/nrpe_helpers.py

```diff
@@ -905,14 +914,21 @@ def _get_cis_audit_check():
 def is_cis_misconfigured():
     """Check if CIS config is misconfigured.
 
-    Return True if CIS config is invalid, otherwise return False.
+    Return True and a message if CIS config is invalid, otherwise
+    return False with empty message.
     """
     if hookenv.config("cis_audit_profile") and hookenv.config(
         "cis_audit_tailoring_file"
     ):
-        return True
+        return (
+            True,
+            "You cannot provide both cis_audit_profile and cis_audit_tailoring_file",
+        )
+
+    if hookenv.config("cis_audit_profile") not in VALID_CHARM_CIS_PROFILES:
```

Comment:
Since the `cis_audit_profile` config directs the user to this latest docs: https://ubuntu.com/security/certifications/docs/usg/cis/audit, it's not difficult for the user to [find out](https://ubuntu.com/security/certifications/docs/usg/cis/compliance) the profile are now prefixed by `cis_`, and they might be wondering why this charm is still using the old profile options (e.g. is the charm still using older version of `usg` etc).

I think it will be good to explain in the `config.yaml` that for example `cis_level1_workstation`  == `level1_workstation` internally in the charm